### PR TITLE
Charts update

### PR DESCRIPTION
**Pull request by MegaByte875**

### CODE REVIEW

File: charts/nebula-operator/values.yaml [hunk 1]
```diff
@@ -34,8 +34,35 @@ scheduler:
...
+watchNamespace:
+
+# The address the metric endpoint binds to. (default ":8080")
+metricsBindAddr: ":8080"
+
+# The address the probe endpoint binds to. (default: ":8081")
+healthProbeBindAddr: ":8081"
+
+# The TCP port the Webhook server binds to. (default 9443)
+webhookBindPort: 9443
```
Comment (veezhang) [inline charts/nebula-operator/values.yaml:59]: Use the same format?

File: doc/user/nebula_cluster_helm_guide.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ The following table lists is the configurable parameters of the chart and their
 | Parameter | Description | Default |
 |:---------|:-----------|:-------|
 | `nameOverride` | Override the name of the chart | `nil` |
-| `nebula.version` | Nebula version | `v2.0.1` |
+| `nebula.version` | Nebula version | `v2.6.1` |
```
Comment (veezhang) [inline doc/user/nebula_cluster_helm_guide.md:78]: Those now added config in values is not contains.

Comment (MegaByte875) [reply doc/user/nebula_cluster_helm_guide.md:78]: I got it

**Review by veezhang — APPROVED** [review]
LGTM

**Pull request merged**